src/limbo/tools/random_generator.hpp

```diff
@@ -54,6 +54,16 @@ namespace limbo {
         }
 
         /// @ingroup tools
+        /// make a 1-D vector from a double (useful when we need to return vectors)
+        Eigen::VectorXd make_vector(double x)
```

Comment:
Oops, no. I was meant to be math.hpp

It should be better now.

On Mon, May 16, 2016 at 7:19 PM Konstantinos Chatzilygeroudis <
notifications@github.com> wrote:

> In src/limbo/tools/random_generator.hpp
> <https://github.com/resibots/limbo/pull/104#discussion_r63389132>:
>
> > @@ -54,6 +54,16 @@ namespace limbo {
> >          }
> >
> >          /// @ingroup tools
> > +        /// make a 1-D vector from a double (useful when we need to return vectors)
> > +        Eigen::VectorXd make_vector(double x)
>
> Does it make sense to have the make_vector function in the
> random_generator file?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/resibots/limbo/pull/104/files/613dec4ce4d302ca73626c92942e830fd856c96c#r63389132>
>